encoders/firebase-encoders-processor/src/main/java/com/google/firebase/encoders/processor/EncodableProcessor.java

```diff
@@ -166,18 +166,21 @@ private void processClass(Element element) {
       String containingClassName,
       Encoder encoder,
       MethodSpec.Builder configureMethod) {
-    Element element = types.asElement(encoder.type());
+    final Element element = types.asElement(encoder.type());
```

Comment:
What's the motivation for `final` here? is it needed?
For context pls see: http://go/java-practices/final#params_locals

Reply:
I actually disagree with the style guide there, but I'll stick with it.